src/main/java/com/exactpro/th2/codec/qfj/QFJCodec.java

```diff
@@ -376,4 +569,16 @@ private int validateTag(int tag, String key, String dictionary) {
         }
         return tag;
     }
+
+    @NotNull
+    @Override
+    public MessageGroup decode(@NotNull MessageGroup messageGroup) {
+        return null;
+    }
+
+    @NotNull
+    @Override
+    public MessageGroup encode(@NotNull MessageGroup messageGroup) {
+        return null;
```

Comment:
I think we should throw exception here instead of returning `null` (especially when method is annotated with `@NotNull`). Or we can call other encode/decode method by supplying dummy reporting context